packages/modals/src/styled/StyledIcon.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import AlertErrorStrokeIcon from '@zendeskgarden/svg-icons/src/16/alert-error-stroke.svg';
+
+export const StyledIcon = styled(AlertErrorStrokeIcon)`
+  position: absolute;
+  top: ${props => props.theme.space.base * 5.5}px;
```

Comment:
I could make that change — what do you suggest? Since `padding-top` is not located in this file, how would the calculation be made? Or would the `top` style be moved out of this file?

Reply:
You would `export` the necessary dimensions from the various locations. Again, might be unneeded overhead. This is actually a valid case for pulling the particular property (i.e. `top`) into the other styled file using `& ${StyledIcon} { ...` so that you can take advantage of the local set of size dimensions.